masterlist.yaml

```diff
@@ -17651,6 +17656,12 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Immersive Citizens - AI Overhaul' ]
         condition: 'active("Immersive Citizens - AI Overhaul.esp") and not active("CFTO - Immersive Citizens Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          -'Creation Club Bittercup'
```

Comment:
Please add an empty space after the `-`.

Reply:
_facepalm_ 

Of course. Seriously, why is programming so damn finnicky?